use_cases/eluc/data/eluc_data.py

```diff
@@ -27,7 +27,13 @@ def encode_as_df(self, df: pd.DataFrame) -> pd.DataFrame:
         new_df = df.copy()
         for col in new_df.columns:
             if col in self.fields:
-                new_df[col] = (new_df[col] - self.fields[col]["range"][0]) / (self.fields[col]["range"][1] - self.fields[col]["range"][0])
+                min_val = self.fields[col]["range"][0]
+                max_val = self.fields[col]["range"][1]
+                # If min and max are the same, just set value to 0
+                if min_val == max_val:
+                    new_df[col] = 0
+                else:
+                    new_df[col] = (new_df[col] - self.fields[col]["range"][0]) / (self.fields[col]["range"][1] - self.fields[col]["range"][0])
```

Comment:
Fixed an edge case where all values in a column are the same. We just scale this to 0, however maybe it's better to scale to 0.5?

Reply:
I think 0 should be fine - the scaler will map it to the min value, which is also the max value.
That being said, if all the values in the column are the same, the column is probably not useful...